src/components/scenes/FioSentRequestDetailsScene.js

```diff
@@ -98,7 +97,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
 }))
 
 const FioSentRequestDetailsScene = connect((state: RootState) => {
-  const wallet = UI_SELECTORS.getSelectedWallet(state)
+  const wallet = state.ui.wallets.byId[state.ui.wallets.selectedWalletId]
```

Comment:
should we add a type here? `const wallet: GuiWallet`

Reply:
Flow is able to infer the type here just fine, since it's just a property lookup off of `RootState`, which has solid type definitions.

Reply:
I put in the requested type definitions when I put back `getSelectedWallet`.